test/move_effect_after_you.c

```diff
@@ -8,7 +8,7 @@ ASSUMPTIONS
 
 DOUBLE_BATTLE_TEST("After You makes the target move after user")
 {
-    if (B_RECALC_TURN_AFTER_ACTIONS >= GEN_8) KNOWN_FAILING; // #2615.
+    if (B_RECALC_TURN_AFTER_ACTIONS >= GEN_8);
```

Comment:
This line could be deleted.

Potentially `ASSUME(B_RECALC_TURN_AFTER_ACTIONS >= GEN_8);` could make sense, meaning "this test only makes sense if `B_RECALC_TURN_AFTER_ACTIONS` is `GEN_8` or newer", but for this specific test I think it should always pass. It's just that the bug only applied when that config is set.

Reply:
Done